spec/param.json

```diff
@@ -28,6 +28,10 @@
             "description": "An optional boolean signifying the parameter should be obscured when displayed.",
             "type": "boolean",
             "default": false
-        }
+        },
```

Comment:
In this file we're providing a programatic view of what the data structure looks like, not an example.